kie-server-parent/kie-server-remote/kie-server-client/src/main/java/org/kie/server/client/ScenarioSimulationServicesClient.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2019 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.kie.server.client;
+
+import java.io.IOException;
+
+import org.kie.server.api.model.ServiceResponse;
+import org.kie.server.api.model.scenariosimulation.ScenarioSimulationResult;
+
+public interface ScenarioSimulationServicesClient {
+
+    /**
+     * Execute given test scenario against specified containerId
+     * @param containerId
+     * @param path
```

Comment:
Please adjust javadoc with description of parameters and return values in this class.

Reply:
Done